src/burn/snd/ymf262.cpp

```diff
@@ -53,6 +53,10 @@ differences between OPL2 and OPL3 shown in datasheets:
 
 */
 
+#if defined (_MSC_VER)
```

Comment:
@dinkc64  I changed this to match the top of YM2612.c as M_PI is undefined in VS otherwise.